_data/merchants.yml

```diff
@@ -118,24 +118,18 @@
       url: http://www.guitartheoryrevolution.info/blog/guitar-theory-revolution-store/
     - name: MyMonero Web-based Wallet
       url: https://mymonero.com
-    - name: Pradeep Atluri, Psychiatrist, New York
-      url: http://dr.mindsci.com/
     - name: Simple, no non-sense hosting
       url: https://rootbox.host/
     - name: Web Developer - Stefanos
       url: http://www.stefanosioannou.com/web-development-monero-accepted
     - name: XMR.to Monero to Bitcoin Payment Service
       url: https://xmr.to
-    - name: algoStrategic - Internet Marketing and Web Development
-      url: https://algostrategic.com
```

Comment:
Also, our site was down for a short while today, but it's back online after resolving a network IP address change.